### PR TITLE
Clean up pylint warnings and errors for podman

### PR DESCRIPTION
**Pull request by jwhonce**
* Some of the pedantic errors were not corrected
* Clean up prep for porting to MacOS and PyPi hosting
* Fix broken unittest

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
I have no idea what a self.id versus a self._id or a self.id_ means.
But as best as I can tell the code looks good.
@baude @TomSweeneyRedHat @mheon PTAL

**Comment by baude** [comment]
bot, retest this please

File: contrib/python/podman/podman/libs/images.py [hunk 1]
```diff
@@ -40,53 +40,53 @@ def create(self, *args, **kwargs):
...
         config['net_mode'] = 'bridge'
         config['network'] = 'bridge'
 
-        logging.debug('Image {}: create config: {}'.format(self.id, config))
+        logging.debug('Image %s: create config: %s', self._id, config)
         with self._client() as podman:
-            id = podman.CreateContainer(config)['container']
-            cntr = podman.GetContainer(id)
-        return Container(self._client, id, cntr['container'])
+            id_ = podman.CreateContainer(config)['container']
```
Comment (TomSweeneyRedHat) [inline contrib/python/podman/podman/libs/images.py]: I'm not a big fan of a dash at the end of a bare variable like this.  Could we at least change this to "ctrid" or some such.  Ditto inputt_ below, perhaps "inputStream".  Granted nothing wrong with it as is, the comment can be ignored, just a sweeney-twitch i thought I'd throw out for your consideration.

Comment (jwhonce) [reply contrib/python/podman/podman/libs/images.py]: @TomSweeneyRedHat ack, I followed the PEP8 recommendation and I'm not sure I like it.

File: contrib/python/podman/podman/libs/containers.py [hunk 2]
```diff
@@ -237,9 +237,9 @@ def delete_stopped(self):
             results = podman.DeleteStoppedContainers()
         return results['containers']
 
-    def get(self, id):
+    def get(self, id_):
```
Comment (haircommander) [inline contrib/python/podman/podman/libs/containers.py]: Is there a reason this is id_ instead of _id?

**Comment by haircommander** [comment]
overall LGTM. I don't understand why the underscores are where they are for some variables, but I trust that it's pylint being a poor program.

**Comment by jwhonce** [comment]
bot, retest this please

**Comment by mheon** [comment]
@jwhonce Tests should be fixed on master, can you rebase?

**Comment by mheon** [comment]
One more comment to be addressed and then I think we can merge

File: contrib/python/podman/podman/libs/containers.py (hunk 2 above)
Comment (jwhonce) [reply contrib/python/podman/podman/libs/containers.py]: According to PEP8:

_id is for "private" variables while id_ is for not overriding a Python keyword.

**Comment by jwhonce** [comment]
@mheon I think we're good to go.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**